tools/tpm2_rsaencrypt.c

```diff
@@ -112,16 +118,11 @@ static bool on_option(char key, char *value) {
     }
         break;
     case 'o': {
-        bool result = files_does_file_exist(optarg);
-        if (result) {
-            return false;
-        }
-        ctx.output_file_path = optarg;
-        ctx.flags.o = 1;
```

Comment:
This is changing some semantics here, I agree with your change but I think it should be commented in the commit message.

Reply:
Doesn't this clarify it, "Make -o option optional and default to stdout. When
printing to stdout, use an xxd compatible hexdump
format."

That's the commit message body.

Reply:
@williamcroberts I didn't mean that part but the fact that now the output file would be overwritten if already existed. But before your change it would fail if the file already existed.